src/mui/layout/withResources.js

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
```

Comment:
Not fan of the HOC. I suggest you move the selector function to the same place as the resources reducer instead.